rails-questions.md

```diff
@@ -2,26 +2,34 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
-
+> rails new 'app_name' -d postgresql -T
 
 2. Explain each section of the following route:  get '/library/:id' => 'book#show'
 
-get -
-library -
-:id -
-book -
-show -
+get - ask router to match request to a controller action
+library - 
+:id - param called id
+book - controller name where request is dispatched to
+show - action "show" or display to be performed by the controller. 
 
-3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
+I missed this class. Need to review it more.
 
+3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
+Destroy the model that was supposed to hold the foreign key and create a new one model with the foreign
+key. This actually happened to my pair in one of the challenges and after struggling to add it for a couple
+of days, we decided to delete it and start all over again. 
 
```

Comment:
If you are just starting off you can delete it for sure. But you can also change your database with a migration. 
$ rails generate migration add_foreign_key
in the migration file:  add_column :table_name, :column_name, :datatype
$ rails db:migrate